README.md

```diff
@@ -12,22 +12,60 @@ This extension enables you to run build validation on a Docs conceptual or Learn
 
 ## How to use the Docs Validation extension
 
+### Full-repository validation
+
 1. Open a Docs conceptual or Learn repo in VS Code with the extension installed.
-1. If you're a Microsoft employee, optionally sign in in by clicking **Docs Validation** from the VS Code status bar. You can still use the extension without signing in, but sign in is recommended to use the most recent build information for private repos.
-1. Trigger a build by clicking **Yes** when prompted after sign in, by clicking **Docs Validation** on the status bar then clicking **Validate** from the drop-down menu, or by right-clicking any file in the repo and selecting **Validate this repository**.
+1. For the first time you use the extension, you will be suggested to choose your user type between **Microsoft employee** and **Public contributor** before you can use full-repository validation.
+1. For **Microsoft employee**, you are required to sign in before using full-repository validation. You can click **Docs Validation** on the status bar then click **Sign in** from the drop-down menu to sign in.
+1. After signing in, you can trigger a validation by clicking **Yes** when prompted. You can also click **Docs Validation** on the status bar then click **Validate** from the drop-down menu, or by right-clicking any file in the repo and selecting **Validate this repository**.
 1. Build validation will run locally and all results will be output to the Problems pane.
 
 ![OAuth](https://github.com/docascode/vscode-docs-build/blob/master/resources/vscode-docs-build.gif?raw=true)
 
 > **Note:** The first time you validate a repo, all the Docs build dependencies will be fetched and cached locally. Subsequent validation runs will be faster.
 
+### Real-time validation
+
+1. Open a Docs conceptual or Learn repo in VS Code with the extension installed.
+2. For the first time you use the extension, you will be suggested to choose your user type between **Microsoft employee** and **Publish contributor** before you can use real-time validation.
+3. For **Microsoft employee**, you are required to sign in before using real-time validation. You can click **Docs Validation** on the status bar then click **Sign in** from the drop-down menu to sign in.
+4. The real-time validation is disabled by default, you can enable it in the extension settings. You will be suggested to reload the extension after you enable real-time validation.
```

Comment:
`You will be asked to reload the extension after you enable real-time validation in the settings.`